### PR TITLE
Paper edits, cached images, and a few final code edits.

### PR DESCRIPTION
**Pull request by janewliang**
Wrote things for the abstract, intro, data, future discussion, normality, and linear regression sections. I only worked on the linear regression and PCA parts of the linear regression section, not Fourier or model selection, so hopefully this won't raise merge conflicts with @benjaminleroy or @hiroto-udagawa if you've also been writing.

Added images used in the slides/paper to the project-level images folder so they are cached. We still need to make sure that they can all be reproduced.

Speaking of which, I broke the scripts to get the PCA plots, so I'm working on fixing that. 

Will create a pull request with the updated READMEs and Makefiles for everything sometime very late today. Or tomorrow morning.

### CODE REVIEW

File: paper/main_sections/regression.tex [hunk 1]
```diff
@@ -1,65 +1,104 @@
...
-the validity of this assumptions, as discussed in the \textit{"Normality Assumptions"} 
-section. As might be expected, we do not know the true value of $\beta$ and as such
-need to estimate, using matrix algebra we get a closed from solution:
+\par It is further assumed that the errors $\epsilon_i$ for each observation 
+$i$ are independent and identically distributed with $N(0, \sigma^2)$, and 
+that the errors are independent of $X$. The coefficients $\beta$ can be 
+estimated with the closed-form solution:
 
 \begin{equation}
 \hat{\beta} =(X^T X)^{-1} X^T
```
Comment (benjaminleroy) [inline paper/main_sections/regression.tex]: I swear I corrected this. Can you make sure it says (X^T X)^-1 X^T y      \* include the y.

Comment (janewliang) [inline paper/main_sections/regression.tex]: Yep.

**Comment by benjaminleroy** [comment]
A lot of the images are old (and wrong, sorry, I have too many images right now, I'll post them soon) \* that's fine tho.

For your new images, the "gray" cmap is ok, but I can't tell what they are with just that picture. Think about changing the labels and titles. ++ did you add the code to make it?

**Comment by janewliang** [comment]
Code to generate the normality slices should be in the final script. I did provide captions to explain them in the paper, but I could also add titles.

**Comment by benjaminleroy** [comment]
maybe color bars for your new charts and then it looks good to merge

**Comment by benjaminleroy** [comment]
Thank you for dealing with all the READMEs btw

**Comment by benjaminleroy** [comment]
yay!

**Pull request merged**